examples/workflows/experimental/new_decorators_basic_script.py

```diff
@@ -0,0 +1,27 @@
+from hera.shared import global_config
+from hera.workflows import RunnerInput, RunnerOutput, WorkflowTemplate
+
+global_config.experimental_features["script_annotations"] = True
+global_config.experimental_features["script_pydantic_io"] = True
+
+w = WorkflowTemplate(name="my-template")
+
+
+class MyInput(RunnerInput):
+    user: str
+
+
+@w.script()
+def hello_world(my_input: MyInput) -> RunnerOutput:
+    output = RunnerOutput()
+    output.result = f"Hello Hera User: {my_input.user}!"
+    return output
+
+
+# Pass script kwargs (including an alternative public template name) in the decorator
+@w.set_entrypoint()
```

Comment:
Can we change this to `w.set_entrypoint` instead?

Alternatively should we just have an entrypoint keyword in w.script?

Reply:
I had some problem using a non-bracketed decorator before, seems fine now though 🤷